schemamd/render.go

```diff
@@ -270,7 +270,7 @@ func writeNestedTypes(w io.Writer, nestedTypes []nestedType) error {
 func writeObjectAttribute(w io.Writer, path []string, att cty.Type, group groupFilter) ([]nestedType, error) {
 	name := path[len(path)-1]
 
-	_, err := io.WriteString(w, "- **"+name+"** (")
+	_, err := io.WriteString(w, "- `"+name+"` (")
```

Comment:
This format seems more consistent with docs online, this change is not too important

Reply:
This was an active choice to move away from code formatting in these cases, but you did point out its inconsistent with the titles even in the generated pages.